include/gatekeeper_net.h

```diff
@@ -405,6 +409,12 @@ ipv6_if_configured(struct gatekeeper_if *iface)
 	return !!(iface->configured_proto & CONFIGURED_IPV6);
 }
 
+static inline int
+get_num_ports(struct gatekeeper_if *iface)
+{
+	return iface->num_ports;
```

Comment:
Write Lua functions to deal with this part of the configuration because at least the GT block will need it as well.

Remember to verify that the back interface is enabled as well in the new function.